bundle/Resources/views/Content/form_fields.html.twig

```diff
@@ -1,39 +1,117 @@
+{# Important overrides to hide labels and errors on some structural fields #}
+
+{%- block ezrepoforms_content_field_row -%}
+    {{ block('form_rows') }}
+{%- endblock ezrepoforms_content_field_row -%}
+
+{%- block _ezrepoforms_user_register_fieldsData_entry_row -%}
+    {% if form.children.value is defined %}
+        {{- form_row(form) -}}
+    {% else %}
+        <div class="ez-field-edit ez-field-edit-{{ form.vars.value.fieldDefinition.fieldTypeIdentifier }}">
+            <div class="ez-field-edit-text-zone">
+                {{ form_label(form) }}
+                {{ form_errors(form) }}
+            </div>
+            <div class="ez-field-edit-ui">
+                <p class="non-editable">
+                    {{ "content.field.non_editable"|trans({}, "ezrepoforms_content") }}
+                </p>
+            </div>
+        </div>
+        {%- do form.setRendered() -%}
+    {% endif %}
+{%- endblock _ezrepoforms_user_register_fieldsData_entry_row -%}
+
+{%- block _ezrepoforms_content_edit_fieldsData_widget -%}
+    {% for field in form %}
+        {% if field.value is defined %}
+            {{- form_row(field) -}}
+        {% else %}
+            <div class="ez-field-edit ez-field-edit-{{ field.vars.value.fieldDefinition.fieldTypeIdentifier }}">
+                <div class="ez-field-edit-text-zone">
+                    {{ form_label(field) }}
+                    {{ form_errors(field) }}
+                </div>
+                <div class="ez-field-edit-ui">
+                    <p class="non-editable">
+                        {{ "content.field.non_editable"|trans({}, "ezrepoforms_content") }}
+                    </p>
+                </div>
+            </div>
+            {%- do field.setRendered() -%}
+        {% endif %}
+    {% endfor %}
+{%- endblock _ezrepoforms_content_edit_fieldsData_widget -%}
+
+{# General form theming #}
+
+{%- block form_row -%}
+    {% set is_fieldtype = form.parent.vars.value.fieldDefinition is defined %}
+    {% set class_prefix = 'field' %}
+
+    {% if is_fieldtype is same as(false) %}
+        {% set class_prefix = 'sub-field' %}
+    {% endif %}
+
+    {%- set row_classes -%}
+        {% spaceless %}
+            ez-{{ class_prefix }}-edit
+            {%- if is_fieldtype %} ez-field-edit-{{ form.parent.vars.value.fieldDefinition.fieldTypeIdentifier }}{% endif -%}
+            {%- if required %} ez-{{ class_prefix }}-edit-required{% endif -%}
+            {%- if disabled %} ez-{{ class_prefix }}-edit-disabled{% endif -%}
+        {% endspaceless %}
+    {%- endset -%}
+
+    <div class="{{ row_classes }}">
+        <div class="ez-{{ class_prefix }}-edit-text-zone">
+            {{ form_label(form) }}
+            {{ form_errors(form) }}
+        </div>
+        <div class="ez-{{ class_prefix }}-edit-ui">
+            {{ form_widget(form) }}
+        </div>
+    </div>
+{%- endblock form_row -%}
+
+{%- block form_label -%}
+    {# Based on default Symfony label. Removed "required" class. #}
```

Comment:
is removing the `required` class the only difference ? if so I think we can live with this `required` class to avoid duplicating code

Reply:
Yes, it was the only difference. Let's leave `required` class on `<label></label>` then.